### ISSUE TITLE
Means of communication

### ISSUE THREAD

**Issue by gedankenstuecke** [gedankenstuecke/snpr#195]
So far we are not really standardized in our means of communication. We are using Twitter, eMails off the mailinglist, eMails on the mailing list (not so much recently) and what not. This makes it rather hard to follow if you are new to the project. 

Should we somewhat standardize this? And if so: To one of the means mentioned or something else?

**Comment by philippbayer** [comment]
I think we should put a bit into the README in the form of a "How to contribute" paragraph, we don't have to make it too professional (yet?). In there we list the two main channel of communications, github issues and the snpr-dev mailing list.

I like to keep github issues to a technical level - each comment should detail the issue in question and keep to the parental issue. I feel like we should use snpr-development for more general discussions, stuff on deployments, technicalities that aren't bound to issues, discussions that are allowed to go walkabout, everything.

**Comment by gedankenstuecke** [comment]
I agree that it doesn't have to be too professional so far. But I think we are having two things going at the same time: Development of the software & actually running the software. And both can't be kept separate at any time, so we should find a way to deal with this in a sensible way. Maybe we should just make sure we use the mailing list more frequently?

**Comment by tsujigiri** [comment]
Discussing things here seems more transparent than the mailing list. I don't mind having less technical discussions here as well. On the other hand, there might be people not on Github, who want to follow discussions via the mailing list. So, maybe we should think about whether something is interesting to non-Github users as well and post accordingly?

**Comment by gedankenstuecke** [comment]
I think in terms of transparency it doesn't make too much of a difference and in principle I have no objections to discuss non-technical stuff via issues as well. But I see your point re: being not on GitHub but being interested in the project. Maybe we do a dedicated mailinglist for actually "running" openSNP, which can then include technicalities like which servers to rent, deployment but also "should we do crowdfunding?" etc?

**Comment by tsujigiri** [comment]
For the technical things it might bring people one step closer to contributing, when they are forced to create a Github account in order to follow technical issues, though. ;) But yeah, mailing lists are probably the least involving thing for everyone. So, let's just use it more? snpr-dev already kind of is the "technical" version, according to the name, right? So, do we need another one or is one enough for everyone?

**Comment by gedankenstuecke** [comment]
I think one is enough, but I changed the group name to "openSNP: development & operation", the email is still the old one, so we don't break anything. :-)
